modules/compliance-operator-install.adoc

```diff
@@ -25,4 +25,11 @@ For more information about restarting Sensor, see "Restarting Sensor in the secu
 
 . Select the *Compliance Operator* to view the details page.
 
-. Read the information about the Operator, and then click *Install*.
\ No newline at end of file
+. Read the information about the Operator, and then click *Install*.
+
+[IMPORTANT]
+====
+If you are using the compliance feature, you can schedule your scan by using {product-title-short} to create a compliance scan schedule instead of creating the `ScanSettingBinding` on the Compliance Operator.
+
+For more information about scheduling a compliance scan by using the compliance feature, see "Customizing and automating your compliance scans".
+====
```

Comment:
I think it would be nice if we could really make this stand out more and differentiate it from the next section.  If you go this path, then DO NOT follow the steps below sort of thing.